packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_ExpressionValue.res

```diff
@@ -5,26 +5,33 @@
 module Extra_Array = Reducer_Extra_Array
 module ErrorValue = Reducer_ErrorValue
 
+@genType.opaque
+type internalCode = Object
+
 @genType
 type rec expressionValue =
   | EvArray(array<expressionValue>)
   | EvBool(bool)
   | EvCall(string) // External function call
   | EvDistribution(DistributionTypes.genericDist)
+  | EvLambda((array<string>, internalCode))
```

Comment:
I don't understand why this is `internalCode` and not `expressionValue`.

Reply:
Because a lamda expression is not simply a value. It is an expression (thus compiled code) with parameters.

Reply:
That makes sense to me. I suggest leaving more documentation to explain this, somewhere ideally in the codebase. Maybe a README.md in the reducer directory.